src/smart-components/role/RolePermissionsTable.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Access, Role } from '../../redux/reducers/role-reducer';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+
+interface RolePermissionsTableProps {
+  viewedRole?: Role;
+}
+
+export const RolePermissionsTable: React.FunctionComponent<RolePermissionsTableProps> = ({ viewedRole }) => {
+  const COLUMNS: string[] = ['Application', 'Resource type', 'Operation'];
```

Comment:
```suggestion


const COLUMNS: string[] = ['User group', 'Workspace assignment (TBD)'];

export const RolePermissionsTable: React.FunctionComponent<RolePermissionsTableProps> = ({ viewedRole }) => {
```
Also the string should be localized